owncloud/owncloud.py

```diff
@@ -794,16 +1044,69 @@ def delete_attribute(self, app, key):
         """
         path = 'deleteattribute/' + urllib.quote(app, '') + '/' + urllib.quote(self.__encode_string(key), '')
         res = self.__make_ocs_request(
-                'POST',
-                self.OCS_SERVICE_PRIVATEDATA,
-                path
-                )
+            'POST',
+            self.OCS_SERVICE_PRIVATEDATA,
+            path
+        )
         if res.status_code == 200:
             tree = ET.fromstring(res.content)
             self.__check_ocs_status(tree)
             return True
         raise ResponseError(res)
 
+    def get_apps(self):
+        """ List all enabled apps through the provisioning api.
+        Returns a dict of apps, with values True/False, representing the enabled state.
```

Comment:
Please use the usual docstring format for return value.
Also add the usual ':raises: ResponseError`